@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -145,6 +145,14 @@ export function overlayOnElevation(tile: Tile): void {
         return;
     }
 
+    // Refine tile elevation range, which is initially set to the broader range given by
```

Comment:
because you are changing the Tile, please add this to the documentation, otherwise the side affect may not be apparent to users of this method.